analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Loops.cs

```diff
@@ -61,7 +61,9 @@ public void Loops_InstructionVisitedMaxTwice_For_FixedCount(string loop)
             """;
         var validator = SETestContext.CreateCS(code, "int arg", new AddConstraintOnInvocationCheck()).Validator;
         validator.ValidateExitReachCount(1);
-        validator.TagValue("End").Should().HaveOnlyConstraints(ObjectConstraint.NotNull, TestConstraint.First);   // Loop was entered, arg has only it's final constraints after looping once
+        validator.TagValues("End").Should().SatisfyRespectively(    // Loop was entered, arg has only it's final constraints after looping once or twice
```

Comment:
The comment is confusing. Is it once, or is it twice?

Reply:
The comments are adding more confusion now, because their original purpose meaning got removed. Just remove all 
`Loop was entered, arg has only its final constraints`
`loop once`
`loop twice`
and it's going to be clear enough when there's `SatisfyRespectively`